secrets.yaml

```diff
@@ -0,0 +1,7 @@
+secrets:
+  - profileName: 'wifiWPA2'
```

Comment:
These profile names should match the ones in the config file, but it's not clear whether they should refer to the Wi-Fi profile name or IEEE. It would be easier to figure out if we match the profile name in our example.